cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -1532,3 +1532,24 @@ declare var WebSocket: {
 };
 
 type BinaryType = "arraybuffer" | "blob";
+
+/**
+ * Shows the given message and waits for the enter key pressed.
+ * @param message
+ */
+declare function alert(message?: string): void;
+
+/**
+ * Shows the given message and waits for the answer. Returns the user's answer as boolean.
+ * @param message
+ */
+declare function confirm(message?: string): boolean;
+
+/**
+ * Shows the given message and waits for the user's input. Returns the user's input as string.
+ * If the default value is given and the user inputs the empty string, then it returns the given
+ * default value.
+ * @param message
+ * @param defaultValue
+ */
+declare function prompt(message?: string, defaultValue?: string): string;
```

Comment:
```suggestion
declare function prompt(message?: string, defaultValue?: string): string | null;
```
Matches `lib.dom.d.ts`.